README.txt

```diff
@@ -706,6 +706,260 @@
 
             python2.7 python/computeEffCurves.py --file=config/configEffPredictor.cfg --deubg=True
 
+
+	# 3.c.iii Analysis of QC data stored in an excel file
```

Comment:
Let's change the title here.  You're not really analyzing the data found in these excel files, you're plotting it.  The README should reflect this.

Additionally while these are python scripts that you have written it makes more sense to structure this under `Section 3.b. genericPlotter` after the first helper script subsection.  e.g.

```
          # 3.b.ii  Helper Script - Parsing Excel Files to genericPlotter
```

Let's move this all to start after Line #617 but before Line #618.